deploy/constants/constants.ts

```diff
@@ -0,0 +1,69 @@
+import { ChainId, parseAmount } from '@angleprotocol/sdk/dist';
+import { BigNumber } from 'ethers';
+import { parseEther } from 'ethers/lib/utils';
+// Mined address for the stablecoin
+export const minedAddress = '0x0000206329b97DB379d5E1Bf586BbDB969C63274';
+export const stableName = 'USD';
+export const vaultsList = ['wstETH'];
+export const forkedChain = ChainId.MAINNET;
+export const forkedChainName = 'mainnet';
+
+export const immutableCreate2Factory = '0x0000000000FFe8B47B3e2130213B802212439497';
+
+export const OFTs: OFTsStructure = {
```

Comment:
We can do a call to our sdk for that

Reply:
We can, only reason why I'm fine not doing it now is because when you deploy it's often a pain to have to wait for the SDK update to get all the addresses here

Reply:
I am not sure I understand your point?